### ISSUE TITLE
Split payment

### ISSUE THREAD

**Issue by denny7** [stripe/stripe-react-native#474]
**Is your feature request related to a problem? Please describe.**

I would like to make split payment between 2 credit cards in my RN app after migrate from `tipsi-stripe`.
What I want is to have 2 different `CardField` components on the same page. After PAY button is clicked I would like to call `confirmPayment` for both payments. I have 2 clientSecrets from backend and  call `initPaymentSheet` for each of them. The problem comes from  that I could't distinguish credit card data when I call `confirmPayment`. The second `CardField` component seems to override first  `CardField` component and both calls of `confirmPayment` sends second credit card data. 

**Describe the solution you'd like**

I would like to implement `CardField` with some identifier and later pass this identifier to `confirmPayment` so in that way I will have the ability to have 2 different `CardField` components with 2 different payment intents.

**Comment by thorsten-stripe** [comment]
You can only have one `CardField` mounted at a time. I'd recommend you collect card details one after the other. For more help with your specific integration please use our developer chat https://stripe.com/go/developer-chat

**Comment by denny7** [comment]
Hi @thorsten-stripe. Thanks for answer.

I've reworked my Payment logic and now I've got different screens for both credit cards.
Now I have problem with second card again. When I try to call `confirmPayment` it gives me this response:

```
"error": {
    "code": "Failed",
    "declineCode": null,
    "localizedMessage": "Card details not complete",
    "message": "Card details not complete", 
    "stripeErrorCode": null, 
    "type": null
}
```

The screens for the first and the second credit cards are almost same. The first one was processed successfully but the second throws this error. I've collected the data from `CardField` and everything seems to be OK (the problem with override first card has gone).
This is the data of the credit card:
```
 {
     "brand": "Visa",
     "complete": true,
     "expiryMonth": 11,
     "expiryYear": 33,
     "last4": "1111",
     "name": "John Doe"
}
```

What could be the problem?
